UI/window-projector.cpp

```diff
@@ -31,6 +31,12 @@ OBSProjector::OBSProjector(QWidget *widget, obs_source_t *source_, int monitor,
 	if (isAlwaysOnTop)
 		setWindowFlags(Qt::WindowStaysOnTopHint);
 
+	hideFrame = config_get_bool(GetGlobalConfig(), "BasicWindow",
+				    "HideProjectorFrame");
+
+	if (hideFrame)
+		setWindowFlags(Qt::FramelessWindowHint);
```

Comment:
When both always-on-top and hide-frame are set, always-on-top is disabled here.

```suggestion
		setWindowFlag(Qt::FramelessWindowHint, true);
```